examples/PyGame/dynamics_adaptation_sliding.py

```diff
@@ -31,6 +32,7 @@
     n_output=robot_config.N_JOINTS,
     pes_learning_rate=1e-2, backend='nengo')
```

Comment:
might want to try a lower learning rate, depending on where the target is the arm can become unstable

Reply:
noted! will reduce